translations/en.json

```diff
@@ -54,6 +68,21 @@
         "description6": "Use the “Edit” mode to add any personal note to a contact.",
         "description7": "Create a group of contacts by clicking the “NEW” button.",
         "description8": "Select a contact or a group of contacts to send shared emails.",
+        "distributionList": {
+            "label": {
+                "filterHint": "You can filter this list by looking for specific member’s name.",
+                "manager_total": "Manager list {{total}}",
+                "member_total": "Member list {{total}}"
+            },
+            "placeholder": {
+                "filter_member": "Filter an address"
+            }
+        },
+        "distributionLists": {
+            "label": {
+                "description": "Description"
+            }
+        },
```

Comment:
Remove these strings, they are relative to the distribution lists, and we changed them a bit in our branch